packages/marketplace/src/components/ui/submit-app-wizard/steps/step-create-new-app.tsx

```diff
@@ -4,14 +4,20 @@ import { WizardStepComponent, SetWizardStep } from '../types'
 import { useFormikContext } from 'formik'
 import { formFields } from '../form-fields'
 
-const { authFlowField } = formFields
+const { authFlowField, directApiField } = formFields
 
 export const onPrev = (setWizardStep: SetWizardStep) => () => {
   setWizardStep('INPUT_APP_NAME')
 }
 
+export const onLaunch = (setWizardStep: SetWizardStep, setFieldValue) => () => {
+  setFieldValue(directApiField.name, true)
+  setWizardStep('INPUT_ATHENTICATION_TYPE')
+}
+
 export const onLauchWithinAgencyCloud = (setWizardStep: SetWizardStep, setFieldValue) => () => {
-  setFieldValue(authFlowField.name, 'Authorisation Code')
+  setFieldValue(authFlowField.name, 'authorisationCode')
```

Comment:
We should import `@constants/app-auth-flow`
and use `authFlows.USER_SESSION` instead of `'authorisationCode'`